### ISSUE TITLE
Unable to Zoom/Change Text Size in Mavensmate App

### ISSUE THREAD

**Issue by thegogz** [joeferraro/MavensMate-Desktop#27]
Ok so this is not a complete blocker due to being able to force the app to open the UI in the browser but is going to become an issue in time and I don't see why it can't be done considering similarly built apps (Atom, Slack etc) all support this. 

I have a visual impairment and one of the reasons I heavily adopted MavensMate for Sublime is I can modify the text size in Sublime easily and manage everything through keyboard shortcuts, when a UI for mavensmate opens in the Browser I can use the native zoom in the browser to work with the UI more comfortably. However with the mavensmate-app even though it is displaying a html/web ui the native Zoom has been disabled and there is no option to increase the font size which makes the app impossible to use on my laptop (the Linux machine I posted the other issue from). 

This morning when in the office and on my windows desktop, I found the option in the settings to use the browser as the UI again which is great, I'd prefer to use the dedicated app as I can see the use case for it increasing over time. 

tl;dr

Allow Increasing the Font/Zoom in the dedicated app in the same way slack and atom do.

**Comment by thegogz** [comment]
@joeferraro has this been implemented or why are you closing this? If its a resource thing let me know and I'll see if I can work out how to do it myself and send in a PR.

**Comment by joeferraro** [comment]
was added to linux/windows in 0.0.11-beta.6 (forthcoming) and in osx in 0.0.11-beta.5

**Comment by thegogz** [comment]
Sweet thanks Joe :)
